### PR TITLE
Add test coverage for mouse_tclick testapi function

### PR DESCRIPTION
**Pull request by KevinKipkorir**
Add subtest for assert_and_dclick in the t/03-testapi.t file.
Add new function for assert_and_dclick in the testapi.pm file.

Fixes: https://progress.opensuse.org/issues/94315

### CODE REVIEW

**Review by okurz — CHANGES_REQUESTED** [review]
https://github.com/os-autoinst/os-autoinst/pull/1700#discussion_r655924353

**Comment by KevinKipkorir** [comment]
> https://github.com/os-autoinst/os-autoinst/pull/1700#discussion_r655924353

I reduced duplication on the file you suggested.Also discovered not extending the file "testapi.pm" after adding the subtest function results to failure on unit tests.

File: testapi.pm [hunk 1]
```diff
@@ -581,6 +581,20 @@ sub click_lastmatch {
...
 
+=head2 assert_and_tclick
+
+  assert_and_tclick($mustmatch [, timeout => $timeout] [, button => $button] [, clicktime => $clicktime ] [, dclick => 1 ] [, mousehide => 1 ]);
+
+Alias for C<assert_and_click> with C<$dclick> set.
+
+=cut
+
+sub assert_and_tclick {
```
Comment (kalikiana) [inline testapi.pm:592]: Maybe there's some confusion here. This looks quite nice but that's not what the ticket asks for 🤓️

The AC is `mouse_tclick is covered by at least one unit test`. We don't want a new API function here, just more test coverage for `mouse_tclick`.

**Comment by KevinKipkorir** [comment]
Accidentally closed it.

**Pull request closed without merging**